website/docs/r/codebuild_project.html.markdown

```diff
@@ -134,6 +134,7 @@ The following arguments are supported:
 `environment_variable` supports the following:
```

Comment:
Any chance you can add an extra CR at the end of this? the docs don't parse properly otherwise.

Reply:
This is done and merged BTW